test/system/170-run-userns.bats

```diff
@@ -133,21 +142,24 @@ EOF
     fi
 }
 
+# bats test_tags=ci:parallel
 @test "podman userns=keep-id" {
     user=$(id -u)
     run_podman run --rm --userns=keep-id $IMAGE id -u
     is "${output}" "$user" "Container should run as the current user"
 }
 
+# bats test_tags=ci:parallel
 @test "podman userns=keep-id in a pod" {
     user=$(id -u)
-    run_podman pod create --userns keep-id
+    run_podman pod create --name p_$(safename) --userns keep-id
     pid=$output
     run_podman run --rm --pod $pid $IMAGE id -u
     is "${output}" "$user" "Container should run as the current user"
-    run_podman rmi -f $(pause_image)
+    run_podman pod rm $pid
 }
 
+# FIXME: cannot be parallelized, some sort of "not enough IDs" error
```

Comment:
all test using some form of userns=auto cannot be parallelized as they need a free id range or they just fail, when running many containers in parallel the id range is quickly exhausted in the rootless case as we usually only have 65536 ids there

Reply:
Can we test parallel in rootful mode only for --userns?

Reply:
It might be possible to set up a new bats tag, like `ci:parallel:butonlywhenrunningasroot` but I hesitate to go down that route because it would require complicating the Makefile invocation. Is there a strong argument for pursuing this?